carina-webdriver/src/main/java/com/qaprosoft/carina/core/foundation/webdriver/Screenshot.java

```diff
@@ -296,7 +306,7 @@ private static String capture(WebDriver driver, boolean isTakeScreenshot, String
                 //hotfix to converting proxy into the valid driver
                 if (driver instanceof Proxy) {
     				try {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.53.207.202/sonarqube/coding_rules#rule_key=squid%3AS134)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this nested try block into a separate method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.53.207.202/sonarqube/coding_rules#rule_key=squid%3AS1141)